client/src/actions/apartments.js

```diff
@@ -0,0 +1,7 @@
+import { APARTMENTS_REQUEST } from '../constants/apartments';
+
+export default function fetchApartmentsRequest() {
+  return {
+    type: APARTMENTS_REQUEST
+  };
+}
```

Comment:
Do you have a benefit in mind for avoiding es6 syntax here, e.g. `const fetchAparmentsRequest = () =>({ type: APARTMENTS_REQUEST })`? I guess your way is more future-proofed if we need to do any minor work inside the function before returning?